system-test/fixtures/google-gax-packaging-test-app/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "google-gax-packaging-test-app",
+  "version": "0.0.1",
+  "description": "Test application for google-gax",
+  "main": "src/index.js",
+  "files": [
+    "src",
+    "protos"
+  ],
+  "license": "Apache-2.0",
+  "keywords": [],
+  "scripts": {
+    "lint": "eslint src/ samples/ system-test/ test/",
```

Comment:
For both of these, I've had a lot better luck just doing `eslint '**/*.js'`.

Reply:
Fixed!